Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -979,6 +1048,36 @@ def analyze_premium_ip_score(self, client: Client, ip: str, base_score: int) ->
 
 
 # region Helper functions
+
+def create_relationships(entity_a: str, entity_a_type: str, relationships_response: dict, reliability):
+    """
+    Create a list of entityRelationship object from the api result
+    entity_a: (str) - source of the relationship
+    entity_a_type: (str) - type of the source of the relationship
+    relationships_response: (dict) - the relationship response from the api
+    reliability: The reliability of the source.
+
+    Returns a list of EntityRelationship objects.
+    """
+    relationships_list: List[EntityRelationship] = []
+    for relationship_type, relationship_type_raw in relationships_response.items():
+
+        relationships_data = relationship_type_raw.get('data', [])
+        if isinstance(relationships_data, dict):
+            relationships_data = [relationships_data]
+
+        for relation in relationships_data:
+            name = RELATIONSHIP_TYPE.get(entity_a_type.lower(), {}).get(relationship_type)
+            entity_b = relation.get('id', '')
```

Comment:
Why using the default `''`? just use `[]` and raise exception if not exists.
Do not keep running on the function if it does not exists. It's a bad practice that leads to a lot of bugs that are hard to catch. use [] and print the exception to the debug log.

Reply:
We do want to keep running in the function if it doesnt exist, as the next one might exist.
If one entity_b doesnt exist it doesnt mean we want to raise any errors, just continue and create the other relationships .